packager/react-packager/src/DependencyResolver/index.js

```diff
@@ -186,7 +186,7 @@ HasteDependencyResolver.prototype.getDebugInfo = function() {
 function defineModuleCode({moduleName, code, deps}) {
   return [
     `__d(`,
-    `'${moduleName}',`,
+    `'${moduleName}',`.replace(replacePatterns.BACKSLASHES_RE, '\\\\'),
```

Comment:
can you put the replace() call inside of the `${}` so it explicitly escapes just the module name?